src/main/thrift/parquet.thrift

```diff
@@ -1211,6 +1316,16 @@ struct ColumnIndex {
     * Same as repetition_level_histograms except for definitions levels.
     **/
    7: optional list<i64> definition_level_histograms;
+
+   /** 
+    * A list containing the number of NaN values for each page. Only present
+    * for columns of physical type FLOAT or DOUBLE, or logical type FLOAT16.
+    * If this field is not present, readers MUST assume that there might or
+    * might not be NaN values in any page, as NaNs should not be included
+    * in min_values or max_values.
```

Comment:
```suggestion
    * might not be NaN values in any page, as NaNs are not included
    * in min_values or max_values.
```

Reply:
This looks imprecise to me. NaNs are included in min_values/max_values when IEEE754TotalOrder is used. So perhaps we should simply say that `If this field is not present, readers MUST assume that there might be NaN values in any page`?